definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType.json

```diff
@@ -53,12 +53,6 @@
     "UserRole": "caseworker-divorce-pcqextractor",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "16/05/2019",
-    "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
-    "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
-  },
```

Comment:
This removal causes errors: 
`UserRole authorisations for CaseState
       should allow minimum R access for all Case States per User Role 
         (non-prod)/(prod):
`

For this reason, ill move it to R rather then remove the field